app/controllers/admin/application_controller.rb

```diff
@@ -19,7 +19,8 @@ class ApplicationController < ApplicationController
       podcasts: "https://admin.forem.com/docs/advanced-customization/content-manager/podcasts",
       settings: "https://admin.forem.com/docs/advanced-customization/config",
       tags: "https://admin.forem.com/docs/forem-basics/tags",
-      users: "https://admin.forem.com/docs/forem-basics/user-roles"
+      users: "https://admin.forem.com/docs/forem-basics/user-roles",
+      creator_settings: "https://admin.forem.com/docs/getting-started/first-user-registration"
```

Comment:
This documentation page will have to be entirely revamped once the Creator Onboarding flow is shipped. Though, I'm happy to start adjusting this now as we see fit if need be. 😊

Reply:
I think we can put in the link as this feature is behind a Feature Flag anyways.

In the meantime, let's kickstart the procedure outlined [here](https://forem-team.slack.com/archives/C01S4QZUDNC/p1629749651046900). i.e. adding an issue using the “changes requested” issue template for #success. 

Before we go live, we will just ensure that the documentation has been updated as well.

Reply:
Got it, @Ridhwana! Is there a best practice for putting this behind a feature flag considering it is part of a constant?

Reply:
@juliannatetreault this constant won't need to be put behind a feature flag, because the view that it appears on is already behind a Feature Flag. We can leave it as it :)

Reply:
Okay, perfect, @Ridhwana! I realized last night before I fell asleep that I may have read your message wrong, but I no longer have GitHub on my phone and couldn't amend my comment. 😅

Reply:
Update: I've added an [issue](https://github.com/forem/admin-docs/issues/23#issue-1051055390) for the need for updated documentation around this flow!